presto-spark-base/src/main/java/com/facebook/presto/spark/execution/PrestoSparkTaskExecutorFactory.java

```diff
@@ -300,6 +307,7 @@ public PrestoSparkTaskExecutorFactory(
         this.tempStorageManager = requireNonNull(tempStorageManager, "tempStorageManager is null");
         this.storageBasedBroadcastJoinStorage = requireNonNull(storageBasedBroadcastJoinStorage, "storageBasedBroadcastJoinStorage is null");
         this.prestoSparkBroadcastTableCacheManager = requireNonNull(prestoSparkBroadcastTableCacheManager, "prestoSparkBroadcastTableCacheManager is null");
+        this.memoryRevocationExecutor = newCachedThreadPool(threadsNamed("memory-revocation"));
```

Comment:
Can we inject this in the `PrestoSparkModule` like other executors?




Also, move the assignment closer to other executor variables to keep the same order of declaration. We sort of picky on constructors lol.

```java
class T {

  var a;
  var b;
  var c;


  T (a, b, c)
  {
    this.a = a;
    this.b = b;
    this.c = c;
  }

}